tests/jballerina-unit-test/src/test/resources/test-src/error/stacktrace-project/modules/stacktrace/stacktrace.bal

```diff
@@ -0,0 +1,4 @@
+public function getStackTrace() returns error:CallStackElement[] {
```

Comment:
Shall we a dd the licence header? Check wherever applicable.

Reply:
Added with the commit https://github.com/ballerina-platform/ballerina-lang/pull/30998/commits/3876b1f25a63e9b504b5994027a1d6ef5f992859